apps/emqx_modules/src/emqx_event_message.erl

```diff
@@ -264,3 +265,6 @@ ignore_sys_message(#message{flags = Flags}) ->
 publish_event_msg(Topic, Payload) ->
     _ = emqx_broker:safe_publish(make_msg(Topic, emqx_json:encode(Payload))),
     ok.
+
+remove_header(Message) ->
```

Comment:
Please help to rename this function to `remove_internal_headers`

Reply:
It has been modified and committed